src/main/java/org/glassfish/enterprise/concurrent/ManagedExecutorServiceImpl.java

```diff
@@ -42,16 +42,11 @@
 
 package org.glassfish.enterprise.concurrent;
 
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.LinkedBlockingQueue;
-import java.util.concurrent.RejectedExecutionHandler;
-import java.util.concurrent.SynchronousQueue;
-import java.util.concurrent.ThreadFactory;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;

```

Comment:
Can you instead of using the wildcard .* leave each of the imports?

Reply:
Done!